presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -137,6 +137,7 @@
     public static final String INDEX_LOADER_TIMEOUT = "index_loader_timeout";
     public static final String OPTIMIZED_REPARTITIONING_ENABLED = "optimized_repartitioning";
     public static final String AGGREGATION_PARTITIONING_MERGING_STRATEGY = "aggregation_partitioning_merging_strategy";
+    public static final String LIST_NON_BUILT_IN_FUNCTIONS = "list_non_built_in_functions";
```

Comment:
Let's call it `list_all_functions`?

Reply:
What about `list_sql_functions`, `list_user_defined_functions`, or `list_udfs` if you think the current name is cumbersome.

For people who don't have context, it's not clear from the name what the difference is between "all functions" and "not all functions", and those 3 names at least give some hint.

Reply:
But is this only going to list sql functions? it will list all functions right? Unless you are planning to make it an enum and allow only listing certain functions. Another way to configure this is to have it as `list_static_functions_only`.

Reply:
Since we're using "built in function" as the language in our code, I'm updating the PR to use `list_built_in_functions_only`.